rest/src/main/scala/tamer/rest/TamerRestJob.scala

```diff
@@ -93,7 +92,8 @@ object TamerRestJob {
       offsetParameterName: String = "page",
       increment: Int = 1,
       fixedPageElementCount: Option[Int] = None,
-      authenticationMethod: Option[Authentication[R]] = None
+      authenticationMethod: Option[Authentication[R]] = None,
+      readRequestTimeout: zio.duration.Duration = 30.seconds
```

Comment:
should we take scala.concurrent.duration.Duration or FiniteDuration directly? what's the value added of taking a zio.duration.Duration here?

Reply:
none, durations are a bit all over the place. In this case I was just using the same as the other args.